_aggregations/metric/average.md

```diff
@@ -9,9 +9,20 @@ redirect_from:
 
 # Average aggregations
 
-The `avg` metric is a single-value metric aggregations that returns the average value of a field.
+The `avg` metric is a single-value metric that returns the average value of a field.
 
-The following example calculates the average of the `taxful_total_price` field:
+## Parameters
+
+The `avg` aggregation takes the following parameters: 
+
+| Parameter | Required/Optional | Data type       | Description |
+| :--       | :--               | :--            | :--         |
+| `field`   | Required          | Numeric         | The field for which the average is computed    |
```

Comment:
```suggestion
| `field`   | Required          | Numeric         | The field for which the average is computed.    |
```

Reply:
@kolchfa-aws True in general for the `missing` keyword, but the `avg` aggregation throws an error if the value isn't numeric. Maybe we should state that explicitly?

Also, it would be pretty slick to show an example where the missing values were replaced by the mean of the existing values. Maybe revisit this later?

Reply:
The field's _contents_ need to be numeric. The `field` parameter must contain a string that specifies the field name (as in `"field": "gpa"`: `"gpa"` is a string).